style.md

```diff
@@ -665,13 +665,13 @@ Each type variable is named in one of two styles:
 
 ## Camel case
 
-Sometimes there is more than one reasonable way to convert an English phrase into camel case, such as when acronyms or unusual constructs like "IPv6" of "iOS" are present. To improve predictability, use the following scheme.
+Sometimes there is more than one reasonable way to convert an English phrase into camel case, such as when acronyms or unusual constructs like "IPv6" or "iOS" are present. To improve predictability, use the following scheme.
 
 Beginning with the prose form of the name:
 
- 1. Convert the phrase to plain ASCII and remove any apostrophes. For example, "Müller's algorithm" might become "Muellers algorithm".
+ 1. Convert the phrase to plain ASCII and remove any apostrophes. For example, "Müller's algorithm" might become "Mueller's algorithm".
 
- 2. Divide this result into words, splitting on spaces and any remaining punctuation (typically hyphens).
+ 2. Divide this result into words, splitting into spaces and any remaining punctuation (typically hyphens).
```

Comment:
"Splitting into spaces" does not convey the action that you should take. Furthermore, this wording matches the [Java style guide](https://google.github.io/styleguide/javaguide.html).